client/my-sites/domains/domain-management/subpage-wrapper/index.tsx

```diff
@@ -11,14 +11,15 @@ const SubpageWrapper = ( { children, subpageKey }: SubpageWrapperProps ) => {
 	const subpageParams = getSubpageParams( subpageKey );
 
 	return subpageParams ? (
-		<>
+		<div className="subpage-wrapper">
 			<NavigationHeader
 				navigationItems={ [] }
 				title={ subpageParams.title }
 				subtitle={ subpageParams.subtitle }
+				className="subpage-wrapper__header"
 			/>
-			<div className="subpage-wrapper">{ children }</div>
-		</>
+			<div className="subpage-wrapper__content">{ children }</div>
+		</div>
```

Comment:
@phcp, FYI, I made some changes to the HTML structure of the component.

Reply:
Nice, thank you for letting me know @m1r0! Feel free to apply any improvements/changes you need.